File: software/examples/python/example_output.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-  

HOST = "localhost"
PORT = 4223
UID = "XYZ" # Change to your UID

from tinkerforge.ip_connection import IPConnection
from tinkerforge.bricklet_io16 import IO16

if __name__ == "__main__":
    ipcon = IPConnection() # Create IP connection
    io = IO16(UID, ipcon) # Create device object

    ipcon.connect(HOST, PORT) # Connect to brickd
    # Don't use device before ipcon is connected

    # Set pin 0 on port a to output low
    io.set_port_configuration('a', 1 << 0, 'o', False)

    # Set pin 0 and 7 on port b to output high
    io.set_port_configuration('b', (1 << 0) | (1 << 7), 'o', True)

    raw_input('Press key to exit\n') # Use input() in Python 3

File: software/examples/javascript/ExampleOutput.js
var Tinkerforge = require('tinkerforge');

var HOST = 'localhost';
var PORT = 4223;
var UID = 'gfp';// Change to your UID

var ipcon = new Tinkerforge.IPConnection();// Create IP connection
var io = new Tinkerforge.BrickletIO16(UID, ipcon);// Create device object

ipcon.connect(HOST, PORT,
    function(error) {
        console.log('Error: '+error);        
    }
);// Connect to brickd

// Don't use device before ipcon is connected
ipcon.on(Tinkerforge.IPConnection.CALLBACK_CONNECTED,
    function(connectReason) {
        // Set pin 0 on port a to output low
        io.setPortConfiguration('a', 1 << 0, 'o', false);
        // Set pin 0 and 7 on port b to output high
        io.setPortConfiguration('b', (1 << 0) | (1 << 7), 'o', true);      
    }
);

console.log("Press any key to exit ...");
process.stdin.on('data',
    function(data) {
        ipcon.disconnect();
        process.exit(0);
    }
);

